code/src/Components/FourthQuestion.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Really like this component where you have used the array and then maping! Good work!